$NetBSD: patch-configure,v 1.2 2013/08/07 10:12:43 markd Exp $

Fix
  CCLD     scanpci
../src/.libs/libpciaccess.so: undefined reference to `gzopen'
../src/.libs/libpciaccess.so: undefined reference to `gzclose'
../src/.libs/libpciaccess.so: undefined reference to `gzgets'

--- configure.orig	2013-07-21 09:36:37.000000000 +0000
+++ configure
@@ -18015,10 +18015,10 @@ case $host_os in
 	*netbsd*)
 		case $host in
-		*i386*)
-			PCIACCESS_LIBS="-li386 -lpci"
+		*i[3-9]86*)
+			PCIACCESS_LIBS="$PCIACCESS_LIBS -li386 -lpci"
 			;;
 		*x86_64*|*amd64*)
-			PCIACCESS_LIBS="-lx86_64 -lpci"
+			PCIACCESS_LIBS="$PCIACCESS_LIBS -lx86_64 -lpci"
 			;;
 		esac
 		netbsd=yes
